test/BUILD

```diff
@@ -0,0 +1,105 @@
+cc_library(
+    name = "output_test_helper",
+    srcs = [
+        "output_test.h",
+        "output_test_helper.cc",
+    ],
+    deps = ["//:benchmark"],
+)
+
+cc_binary(
```

Comment:
Is it possible to use cc_test instead?

Shouldn't users be able to do "bazel test :all" to validate that all tests are passing?